### ISSUE TITLE
Hello, can the code of the training part be uploaded?

### ISSUE THREAD

**Issue by ghost** [karoly-hars/3D_scene_rec_CNNs#1]

**Comment by karoly-hars** [comment]
Sorry, the training code will not be included any time soon. I don't have my old code and the re-writing + training + documentation would take a lot of time, which I don't have at the moment.

The training procedure for the depth estimation network is described in:
https://link.springer.com/chapter/10.1007%2F978-3-319-98678-4_38

There is nothing unconventional about training a model for segmentation, so there are lots of good blog posts and example code out there.
In this repo, I used the same network structure for the depth prediction and the segmentation (except the final layer) and the same hyperparameters worked pretty well. If I remember correctly, I used Adam optimizer + I weighted the classes using median frequency weighting (see https://arxiv.org/pdf/1411.4734.pdf) in the loss function. Everything else is pretty standard.

**Comment by ghost** [comment]
Thank u very much！
